src/test/util/net.cpp

```diff
@@ -66,18 +66,7 @@ void ConnmanTestMsg::NodeReceiveMsgBytes(CNode& node, Span<const uint8_t> msg_by
 {
     assert(node.ReceiveMsgBytes(msg_bytes, complete));
     if (complete) {
-        size_t nSizeAdded = 0;
-        for (const auto& msg : node.vRecvMsg) {
-            // vRecvMsg contains only completed CNetMessage
-            // the single possible partially deserialized message are held by TransportDeserializer
-            nSizeAdded += msg.m_raw_message_size;
-        }
-        {
-            LOCK(node.cs_vProcessMsg);
-            node.vProcessMsg.splice(node.vProcessMsg.end(), node.vRecvMsg);
-            node.nProcessQueueSize += nSizeAdded;
-            node.fPauseRecv = node.nProcessQueueSize > nReceiveFloodSize;
-        }
+        node.MarkReceivedMsgsForProcessing(nReceiveFloodSize);
```

Comment:
That is a matter of personal taste (since both are allowed), but I just want to mention here why I, personally, avoid that:
* It is not gdb friendly - you can't set a breakpoint on the `if` body (yes, one can set a breakpoint on the function call, but it becomes tedious if that function is called a lot from other places)
* Creates bigger diff during changes in the future, comapre:

```diff
- if (A) B;
+ if (A) {
+     B;
+     C;
+ }
```

vs

```diff
 if (A) {
     B;
+    C;
 }
```

The latter is easier to read, especially if `A`, `B` and `C` are complex expressions.

Reply:
I agree that it's personal taste and the project style guide allows either.

> It is not gdb friendly - you can't set a breakpoint on the if body (yes, one can set a breakpoint on the function call, but it becomes tedious if that function is called a lot from other places)

Are you familiar with [conditional breakpoints](https://ftp.gnu.org/old-gnu/Manuals/gdb/html_node/gdb_33.html)? You can set the breakpoint to only stop the program if a boolean condition evaluates to true. Here, I think you could add the breakpoint:

`break "net.cpp":68 if complete`

and it would only stop if the if block is going to be called.

Reply:
Alas that does not work if the condition includes function calls.

Reply:
> Alas that does not work if the condition includes function calls.

Why not? From the GDB docs:

> Break conditions can have side effects, and may even call functions in your program.